### PR TITLE
replace tf_conversions with tf2

### PR DESCRIPTION
**Pull request by gleichdick**
tf is deprecated, replaced it with tf2_eigen for noetic

### CODE REVIEW

File: src/rviz_visual_tools.cpp [hunk 1]
```diff
@@ -790,16 +792,16 @@ Eigen::Isometry3d RvizVisualTools::getVectorBetweenPoints(const Eigen::Vector3d&
   // Method 1 - TF - works
```
Comment (gleichdick) [inline src/rviz_visual_tools.cpp:792]: @davetcoleman Why is this workaround needed? Is there a unit test for it?

**Comment by gleichdick** [comment]
Part of the CI fails due to moveit release (ros-planning/moveit#1990)

File: src/rviz_visual_tools.cpp (hunk 1 above)
Comment (davetcoleman) [reply src/rviz_visual_tools.cpp:793]: No, I was in grad school a the time and did not write enough unit tests. This is an embarrassing section of code :-(

You can see a few lines below my attempt to use Eigen instead did not work. I strongly encourage you to remove these comments and just go with which ever one you prefer.

>  Why is this workaround needed? 

On line 814 you see "Method 2" - uncomment that and see if you can get it to work

**Comment by gleichdick** [comment]
Thanks, the "Eigen" solution had just one small issue: `right_axis_vector` was not normalized, because normalize**d**() only returs a copy and does not affect the original variable.
For the testcase, I just threw random numbers into the "TF" solution and used the results. So the expected values are not calculated manually.

**Comment by JafarAbdi** [comment]
Thanks @gleichdick!

**Pull request merged**